components/feature/readerview/src/main/java/mozilla/components/feature/readerview/ReaderViewFeature.kt

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.readerview
+
+import android.support.annotation.VisibleForTesting
+import mozilla.components.browser.session.SelectionAwareSessionObserver
+import mozilla.components.browser.session.SessionManager
+import mozilla.components.concept.engine.Engine
+import mozilla.components.concept.engine.webextension.WebExtension
+import mozilla.components.concept.toolbar.Toolbar
+import mozilla.components.support.base.feature.LifecycleAwareFeature
+import mozilla.components.support.base.log.logger.Logger
+
+class ReaderViewFeature(
+    private val engine: Engine,
+    private val toolbar: Toolbar,
+    sessionManager: SessionManager
+) : SelectionAwareSessionObserver(sessionManager), LifecycleAwareFeature {
```

Comment:
nit: Extends `SelectionAwareSessionObserver` but doesn't use that yet? :)

Reply:
Yes, for now it only registers the observer for the selected session, but there is nothing to observe yet :)